epp-client/src/epp/request.rs

```diff
@@ -5,16 +5,32 @@ pub mod domain;
 pub mod host;
 pub mod message;
 
+use epp_client_macros::*;
+
 use serde::{ser::SerializeStruct, ser::Serializer, Deserialize, Serialize};
 use std::error::Error;
+use std::fmt::Debug;
 use std::time::SystemTime;
 
 use crate::epp::object::{
     ElementName, EmptyTag, EppObject, Extension, Options, ServiceExtension, Services, StringValue,
     StringValueTrait,
 };
-use crate::epp::xml::{EPP_CONTACT_XMLNS, EPP_DOMAIN_XMLNS, EPP_HOST_XMLNS, EPP_LANG, EPP_VERSION};
-use epp_client_macros::*;
+use crate::epp::xml::{
+    EppXml, EPP_CONTACT_XMLNS, EPP_DOMAIN_XMLNS, EPP_HOST_XMLNS, EPP_LANG, EPP_VERSION,
+};
+
+/// Trait to set correct value for xml tags when tags are being generated from generic types
+pub trait EppRequest {
+    type Output: EppXml + Debug;
```

Comment:
Maybe name this `Response` instead of `Output`? Seems easier to follow in this case.

Reply:
Yes, that's more suitable. Thanks.